js/app/controllers/calcontroller.js

```diff
@@ -140,8 +140,10 @@ app.controller('CalController', ['$scope', 'Calendar', 'CalendarService', 'VEven
 				$scope.$apply();
 			});
 		} else {
-			$scope.calendarsPromise = CalendarService.getPublicCalendar(constants.publicSharingToken).then(function(calendar) {
-				$scope.calendars = [calendar];
+			constants.publicSharingToken.split(".").forEach( (token) => {
+				$scope.calendarsPromise = CalendarService.getPublicCalendar(token).then(function(calendar) {
+					$scope.calendars.push(calendar);
+			});
```

Comment:
I'm sorry, but the indentation is still wrong. This should be indented by one tab.